src/main/kotlin/app/configuration/SecureHttpClientProvider.kt

```diff
@@ -13,21 +13,21 @@ import javax.net.ssl.SSLContext
 
 @Component
 @Profile("secureHttpClient")
-class SecureHttpClientProvider: HttpClientProvider {
+class SecureHttpClientProvider : HttpClientProvider {
 
     override fun client(): CloseableHttpClient =
-        HttpClients.custom().run {
-            setDefaultRequestConfig(requestConfig())
-            setSSLSocketFactory(connectionFactory())
-            build()
-        }
+            HttpClients.custom().run {
```

Comment:
I'd expecting this to be an indent of 4 not 8?

Reply:
I have changed  all continuation indentation to be 4 across files